book/src/install.md

```diff
@@ -179,8 +183,8 @@ cd helix
 cargo install --path helix-term --locked
 ```
 
-This command will create the `hx` executable and construct the tree-sitter
-grammars in the local `runtime` folder.
+This command will create the `hx` executable and construct the tree-sitter grammars in the 
+local `runtime` folder.
```

Comment:
```suggestion
This command will create the `hx` executable and construct the tree-sitter
grammars in the local `runtime` folder.
```

(we prefer to avoid cosmetic changes when possible, to keep the diff small)